### PR TITLE
Kogger Sonar

### PR DESCRIPTION
**Pull request by aproof**
Hi!

Our team designs and manufactures sonar modules (https://kogger.tech/).
We try to keep the price as low as possible in order to make such solutions affordable for everyone.

For easy compatibility, the NMEA protocol has been implemented. But the modules support many features that can be found in the open documentation [Open Serial Binary Protocol](https://github.com/koggertech/Kogger-Protocol).

We have prepared the documentation for easy connection and configuration.

Thanks!

### CODE REVIEW

File: common/source/docs/common-underwater-sonars-landingpage.rst [hunk 1]
```diff
@@ -11,6 +11,7 @@ Follow the links below (or in the sidebar) for configuration information based u
 .. toctree::
     :maxdepth: 1
 
+    Kogger Sonar <common-kogger-sonar>
```
Comment (Hwurzburg) [inline common/source/docs/common-underwater-sonars-landingpage.rst:14]: Please place in list in alphabetical order, thanks...will final build test when you believe you are finished...thanks

Comment (aproof) [reply common/source/docs/common-underwater-sonars-landingpage.rst]: Thank you for your comment!

**Comment by Hwurzburg** [comment]
I have fixed some formatting errors and removed duplication of web site urls....and pushed to your branch..will review at next week's wiki dev call..thanks

File: common/source/docs/common-kogger-sonar.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    :target: ../_images/kogger-sonar.png
+
+   
+Recommended Hardware
+--------------------
+- CP210x USB->UART converter and USB OTG cable to configure the module by Android smartphone.
+
+Configuring the echosounder module
+----------------------------------
+The module should be configured before it is connected to the Ardupilot. You need the Android device and GUI installed for configuring the KOGGER Sonar module.
```
Comment (OXINARF) [inline common/source/docs/common-kogger-sonar.rst:20]: Nitpicking: Ardupilot -> ArduPilot

File: common/source/docs/common-kogger-sonar.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+.. image:: ../../../images/kogger-sonar-android-nmea.png
+    :target: ../_images/kogger-sonar-pixhawk.png
+
+.. image:: ../../../images/kogger-sonar-android-flash.png
+    :target: ../_images/kogger-sonar-pixhawk.png
+
+That’s it. The module is ready to use with autopilot.
+
+Connecting to the Ardupilot hardware
```
Comment (OXINARF) [inline common/source/docs/common-kogger-sonar.rst:39]: Same nitpicking.

File: common/source/docs/common-kogger-sonar.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+   
+Recommended Hardware
+--------------------
+- CP210x USB->UART converter and USB OTG cable to configure the module by Android smartphone.
+
+Configuring the echosounder module
+----------------------------------
+The module should be configured before it is connected to the Ardupilot. You need the Android device and GUI installed for configuring the KOGGER Sonar module.
+The GUI is opensource `Kogger Android <https://github.com/koggertech/Kogger-Android/>`__.
+The APK can be downloaded from here: `Kogger Android Releases <https://github.com/koggertech/Kogger-Android/releases/>`__.
```
Comment (OXINARF) [inline common/source/docs/common-kogger-sonar.rst:22]: @aproof can you let us know why the APK isn't in the store? You likely have a good reason, but it's bad practice to direct users to APKs, so I'm just wondering the reason.

File: common/source/docs/common-kogger-sonar.rst (hunk 2 above)
Comment (aproof) [reply common/source/docs/common-kogger-sonar.rst:20]: Thanks!

File: common/source/docs/common-kogger-sonar.rst (hunk 3 above)
Comment (aproof) [reply common/source/docs/common-kogger-sonar.rst:39]: Thanks!

File: common/source/docs/common-kogger-sonar.rst (hunk 4 above)
Comment (aproof) [reply common/source/docs/common-kogger-sonar.rst:22]: You're right! Now we do not have so many resources to do everything in the best possible way. We'll make it. I mean the app store. However, our users already want to get simple operation and visualization of work. Probably should remove this link. What do you think?

**Comment by rmackay9** [comment]
@aproof, it's great to see this new sonar.  I'm sure some users will buy them and put them to good use.

File: common/source/docs/common-kogger-sonar.rst (hunk 2 above)
Comment (aproof) [reply common/source/docs/common-kogger-sonar.rst:20]: image
:)

Comment (OXINARF) [reply common/source/docs/common-kogger-sonar.rst]: Fixed, thanks! 🙂

File: common/source/docs/common-kogger-sonar.rst (hunk 4 above)
Comment (OXINARF) [reply common/source/docs/common-kogger-sonar.rst:22]: Understood. I encourage you to do the work to put it in the store as it gives confidence to users and credibility for you.

Comment (aproof) [reply common/source/docs/common-kogger-sonar.rst:22]: I absolutely agree with you! We’ll do it soon.

**Pull request merged**